server/lib/fusor/engine.rb

```diff
@@ -9,6 +11,9 @@ class Engine < ::Rails::Engine
     config.autoload_paths += Dir["#{config.root}/app/serializers"]
     config.autoload_paths += Dir["#{config.root}/lib/modules"]
 
+    ActiveModelSerializers.config.adapter = :json_api
+    ActiveModelSerializers.config.key_transform = :unaltered
```

Comment:
both of these configurations are important. Default key is dashorized and Sat6 returns snake_case so we want it `unaltered`